src/utils/getTokenList.ts

```diff
@@ -1,12 +1,12 @@
 import { TokenList } from '@uniswap/token-lists'
-import schema from '@uniswap/token-lists/src/tokenlist.schema.json'
-import Ajv from 'ajv'
+// import schema from '@uniswap/token-lists/src/tokenlist.schema.json'
+// import Ajv from 'ajv'
 
 import contenthashToUri from './contenthashToUri'
 import { parseENSAddress } from './parseENSAddress'
 import uriToHttp from './uriToHttp'
 
-const tokenListValidator = new Ajv({ allErrors: true }).compile(schema)
+// const tokenListValidator = new Ajv({ allErrors: false }).compile(schema)
```

Comment:
will we fix this on other PR?

Reply:
Ohh! nice catch I will fix in this one only. I forget to fix it.